src/application/controller/controller.h

```diff
@@ -0,0 +1,58 @@
+#ifndef CONTROLLER_H_
+#define CONTROLLER_H_
+
+#include "FreeRTOS.h"
+#include "state_estimation.h"
+#include <stdbool.h>
+
+/* Enums/Types */
+typedef struct {
```

Comment:
I recommend a union for this. See https://github.com/xioTechnologies/Fusion/blob/main/Fusion/FusionMath.h

Reply:
https://github.com/waterloo-rocketry/cansw_processor_canards/issues/41
im gonna make vectors a common project-wide type